src/worker/webhook.rs

```diff
@@ -0,0 +1,68 @@
+use crate::background_jobs::Environment;
+use crate::models::{CrateOwner, Webhook};
+use crate::schema::{crate_owners, crates, versions, webhooks};
+use diesel::prelude::*;
+use serde_json::json;
+use swirl::PerformError;
+
+// What information do we want to send in the webhook?
+// - NewVersionReleased
+//  - Crate Name
+//  - Crate Version
+//  - Published by
+//  - Published at
+//  -
+
+struct CrateWebhook {
+    name: String,
+    version: String,
+}
+
+#[swirl::background_job]
+pub fn notify_owners(
+    env: &Environment,
+    conn: &PgConnection,
+    krate_id: i32,
+) -> Result<(), PerformError> {
+    println!("Executing notify_owners job!");
+
+    let crate_owners: Vec<i32> = crate_owners::table
+        .select(crate_owners::owner_id)
+        .filter(crate_owners::crate_id.eq(krate_id))
+        .filter(crate_owners::owner_kind.eq(0))
```

Comment:
I'm not sure if you can call filter twice like this in diesel. I think this should be

```suggestion
        .filter(crate_owners::crate_id.eq(krate_id).and(crate_owners::owner_kind.eq(0)))
```